src/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { MultiError } from "./types";
+
+export const getError = (errors: string[]): MultiError =>
+    new MultiError(`Rpc requests unsuccessful.\n${errors.map(err => ` ${err}`).join("\n")}`, errors);
```

Comment:
tiny nit:

prob don't need space prefixing this expression (maybe tab if anything): `` ${err}``